decidim-elections/app/views/decidim/elections/admin/elections/index.html.erb

```diff
@@ -41,11 +41,11 @@
                   <%= icon_link_to "pencil", edit_election_path(election), t("actions.edit", scope: "decidim.elections"), class: "action-icon--edit" %>
                 <% end %>
 
-                <% if allowed_to?(:update, :election, election: election) %>
-                  <% if election.published? %>
-                    <%= icon_link_to "x", url_for(action: :unpublish, id: election, controller: "elections"), "unpublish", class: "action-icon--unpublish", method: :put %>
-                  <% else %>
+                <% if !election.published? %>
```

Comment:
There is a nice tip that RuboCop checks on ruby code (but not on ERB templates): 

```ruby
if [condition]
  do_A
else
  do_B
end
```
is easier to understand than

```ruby
if ![condition]
  do_B
else
  do_A
end
```

So, maybe it would be easier to read in the opposite order, what do you think?

Reply:
Yes! I agree, I can fix it and upload again.